### PR TITLE
validate claimitem payout option vs claim event type

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: application/api/claimitems.go [hunk 1]
```diff
@@ -29,9 +29,10 @@ const (
 type PayoutOption string
 
 const (
-	PayoutOptionRepair      = PayoutOption("Repair")
-	PayoutOptionReplacement = PayoutOption("Replacement")
-	PayoutOptionFMV         = PayoutOption("FMV")
+	PayoutOptionRepair        = PayoutOption("Repair")
+	PayoutOptionReplacement   = PayoutOption("Replacement")
+	PayoutOptionFMV           = PayoutOption("FMV")
+	PayoutOptionFixedFraction = PayoutOption("FixedFraction")
```
Comment (Schparky) [inline application/api/claimitems.go:35]: I realize this was settled this morning, but maybe "Partial" would convey the same meaning in fewer letters/words.

File: application/models/claimitem_test.go [hunk 2]
```diff
@@ -82,11 +163,14 @@ func (ms *ModelSuite) TestClaimItem_Validate() {
 			if tt.wantErr {
 				if vErr.Count() == 0 {
 					t.Errorf("Expected an error, but did not get one")
+					return
```
Comment (Schparky) [inline application/models/claimitem_test.go:166]: All of the if/else conditions are handled and there's nothing after the if, so I don't believe the returns are needed. Is it just for clarity?

**Review by Schparky — APPROVED** [review]

File: application/models/claimitem_test.go (hunk 2 above)
Comment (Baggerone) [reply application/models/claimitem_test.go:166]: Right. Sorry. It looked like it was not exiting the whole test on a failure, and that's what I was trying to accomplish.
I'll see if there is a good way to do that.

File: application/api/claimitems.go (hunk 1 above)
Comment (Baggerone) [reply application/api/claimitems.go:35]: For me "Partial" is a bit ambiguous.

**Pull request merged**